subgraph/query-examples/.graphclientrc.yml

```diff
@@ -0,0 +1,9 @@
+# .graphclientrc.yml
```

Comment:
Should we rename the folder to just `examples`? In the future if we have mutation examples or other things we can put it all under the examples folder.

Reply:
That works. 👍 Inside I'll create a query folder, too.